### PR TITLE
Set Value from Theme

### PR DESCRIPTION
**Pull request by katgirl**
Set font-weight and font-family standalone from theme

### CODE REVIEW

File: files/nutshell/scss/base/_type.scss [hunk 1]
```diff
@@ -136,18 +136,29 @@ html {
...
-	  
+    
     @if $color {
       color: $color;
+    }
+    	  
+    @if $font-weight {
+      font-weight: $font-weight;
+    } @else {
+      font-weight: var(--headings__font-weight);
```
Comment (denniserdmann) [inline files/nutshell/scss/base/_type.scss:153]: I think about removing the fallback, as we already have a fallback in line 116ff. What do you think?

File: files/nutshell/scss/base/_type.scss [hunk 2]
```diff
@@ -136,18 +136,29 @@ html {
...
     }	  
+	  
+    @if $font-family {
+      font-family: $font-family;
+    } @else {
+      font-family: var(--headings__font-family);
```
Comment (denniserdmann) [inline files/nutshell/scss/base/_type.scss:159]: I think about removing the fallback, as we already have a fallback in line 116ff. What do you think?

**Review by denniserdmann — COMMENTED** [review]
Added color option, think about getting rid of fallbacks, see below

File: files/nutshell/scss/base/_type.scss (hunk 1 above)
Comment (katgirl) [reply files/nutshell/scss/base/_type.scss:153]: Sorry, I hadn't seen it at all, that's OK if that one falls on there.

**Pull request merged**